Library/Homebrew/formula_cellar_checks.rb

```diff
@@ -62,8 +62,11 @@ def check_non_libraries
     valid_extensions = %w[.a .dylib .framework .jnilib .la .o .so
                           .jar .prl .pm .sh]
     non_libraries = formula.lib.children.reject do |g|
-      next if g.directory?
-      valid_extensions.include? g.extname
+      if g.directory?
```

Comment:
I considered it, but I don't find `next true` to be very clear, especially for people who are new to Ruby. Glad to change it if you have a strong opinion though.